### PR TITLE
Provided german translation

### PR DESCRIPTION
**Pull request by michelgotta**
Added complete german translation for Balance Manager because it's such a neat tool.

Feel free to send a tip through Gitcoin. 🍾 ✌️

### CODE REVIEW

File: src/languages/index.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import i18next from 'i18next';
 import english from './_english.json';
 import spanish from './_spanish.json';
+import spanish from './_german.json';
```
Comment (samelliott89) [inline src/languages/index.js]: @michelgotta this should be `import german from './_german.json';`

Comment (michelgotta) [reply src/languages/index.js]: @samelliott89 Thanks for the check!

**Comment by ricburton** [comment]
Wow. Thank you!

**Review by ricburton — COMMENTED** [review]
This is really great! As soon as we get the tab changes merged we will merge this.

**Comment by ricburton** [comment]
Sieht gut für mich aus, @jinchung :)

**Review by jinchung — APPROVED** [review]

**Comment by jinchung** [comment]
@michelgotta - thanks for your contribution!

**Comment by ricburton** [comment]
🙌 Preview this pull request.

Built with commit e8f9bf1de59ce381026bc5b6bba10ddee0fc8015

https://deploy-preview-259--balance-manager.netlify.com

**Pull request merged**